recipes/bob.ap/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "2.0.4" %}
+
+package:
+  name: bob.ap
+  version: {{ version }}
+
+source:
+  fn: bob.ap-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.ap/bob.ap-{{ version }}.zip
+  md5: 49f89a84c3c2cb20aa0ffa49d39415a7
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - bob.extension
+    - bob.blitz
+    - bob.core
+    - bob.sp
+    - scipy
```

Comment:
I guess I am still unsure as `bob.blitz` is linking against NumPy. As long as `bob.blitz` is not exposing any NumPy C API or ABI as part of its interface or changing its interface based on NumPy, this should be ok. If you run into issues, just add the `x.x`, bump the build number, and re-render the feedstock with `conda-smithy`.